mbed-trace/mbed_trace.h

```diff
@@ -63,18 +63,18 @@ extern "C" {
 #endif
 
 #ifndef YOTTA_CFG_MBED_TRACE_FEA_IPV6
-#define YOTTA_CFG_MBED_TRACE_FEA_IPV6 1
+#define YOTTA_CFG_MBED_TRACE_FEA_IPV6 0
```

Comment:
All these default values should not be changed here. You should rather set them for example in the cmake configuration file.